CONTRIBUTING.md

```diff
@@ -86,37 +76,46 @@ message that the code in question is special and care should be taken.
   Remember that someone may be looking at your commit in several years time,
   trying to work out the reason for your commit and wondering what on earth you
   were thinking. That someone may be you.
-- Make pull requests a clean representation of the implementation of a feature
-  — meandering commit history is okay for WIP branches if it's supposed to be
-  squashed down to one commit eventually, but development dead-ends and back
-  tracks while developing a feature probably aren't useful to retain. It's okay
-  if this is hard to achieve, but expect people to argue for a squash-merge.
+
+## Pull Requests
+
+- Include a newsfragment before merging. This is a file with a description of
+  your change, to go in the release notes - often less technical than a commit
+  message. This is a file numbered to match the issue or pull request in
+  the `newsfragments` folder - our current list of categories and what to call
+  this can be found [here][news-README]. These are processed on release by the
+  release manager with a tool named [towncrier]. This can be added after you've
+  created the pull request, if you don't have an issue number.
```

Comment:
```suggestion
- Include a newsfragment before merging. This is a file in the
  `newsfragments` folder with a description of your change. Try to make it
  a one-sentence summary aimed at a DIALS user, not a developer. The file is
  numbered to match the issue or pull request and has 
  [one of the allowed extensions from the list][news-README]. 
  If you don't have an issue number you can also add that file after you've
  created your pull request. Newsfragments are processed on release by a
  tool named [towncrier] and make up our release notes. 
```
I really do not like "click [here](https://uxp.de)" links.

Reply:
That's reasonable, but I've also got quite high pressure away from multi-word links (never completely clear if one or more links). I flip-flopped back and forth.